module/apmotel/go.mod

```
@@ -0,0 +1,40 @@
+module go.elastic.co/apm/module/apmotel
```

```suggestion
module go.elastic.co/apm/module/apmotel/v2
```

We version all instrumentation modules the same as the main module, which is currently on v2.